src/app/series/directives/series-feed.component.ts

```diff
@@ -0,0 +1,89 @@
+import { Component, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { DistributionModel, SeriesModel, StoryModel, TabService } from '../../shared';
+
+@Component({
+  template: `
+    <section *ngIf="series && podcast">
+      <div class="hint" *ngIf="noStories">
+        You have no published stories in this podcast.
+      </div>
+
+      <section *ngIf="!noStories">
+        <ul *ngIf="privateStories.length">
+          <h4>Private</h4>
+          <li *ngFor="let s of privateStories">
+            <h5><a [routerLink]="['/story', s.id]">{{s.title}}</a></h5>
+            <p class="private">No publication date set.</p>
+          </li>
+        </ul>
+
+        <ul *ngIf="futurePublicStories.length">
+          <h4>Set for future publication</h4>
+          <li *ngFor="let s of futurePublicStories">
+            <h5><a [routerLink]="['/story', s.id]">{{s.title}}</a></h5>
+            <p class="futurePublic">Will be published {{s.pubDate}}.</p>
+          </li>
+        </ul>
+
+        <ul *ngIf="publicStories.length">
+          <h4>Public feed</h4>
+          <li *ngFor="let s of publicStories">
+            <h5><a [routerLink]="['/story', s.id]">{{s.title}}</a></h5>
+            <p class="public">Published {{s.pubDate}}.</p>
+          </li>
+        </ul>
+      </section>
+    </section>
+  `,
+  styleUrls: ['./series-feed.component.css']
+})
+
+export class SeriesFeedComponent implements OnDestroy {
+
+  noStories: boolean;
+  series: SeriesModel;
+  publicStories: StoryModel[] = [];
+  futurePublicStories: StoryModel[] = [];
+  privateStories: StoryModel[] = [];
+  podcast: DistributionModel;
+  tabSub: Subscription;
+
+  constructor(tab: TabService) {
+    this.noStories = false;
+    this.tabSub = tab.model.subscribe((s: SeriesModel) => {
+      this.series = s;
+      this.podcast = s.distributions.filter(dist => dist.kind === 'podcast')[0];
+      this.sortStories();
+    });
+  }
+
+  sortStories() {
+    this.podcast
+        .parent
+        .followItems('prx:stories', { sorts: 'updated_at:desc' })
```

Comment:
so while walking Luna I realized I could just copy the storyCount-obtaining code from `series.component.ts` in order to get the total # of stories for this new tab -- but looks like it is using count() instead of total() too. 
```
if (series) {
      this.storyCount = series.count('prx:stories');
      this.storyNoun = this.storyCount === 1 ? 'Story' : 'Stories';
} 
```
Does this work for your series with 318 stories in it?